apollo-router/src/query_planner/fetch.rs

```diff
@@ -369,12 +368,11 @@ impl Variables {
 impl FetchNode {
     #[allow(clippy::too_many_arguments)]
     pub(crate) async fn subgraph_fetch(
+        &self,
         service: BoxService,
         subgraph_request: SubgraphRequest,
         service_name: &str,
```

Comment:
Should this use `self.service_name` instead now, like `response_at_path` does? Or can these be intentionally different?